src/node_contextify.cc

```diff
@@ -529,11 +529,13 @@ void ContextifyContext::PropertySetterCallback(
 
   if (ctx->sandbox()->Set(context, property, value).IsNothing()) return;
 
-  Local<Value> desc;
-  if (is_declared_on_sandbox &&
```

Comment:
Instead of dropping the if and replace it by two ifs, another solution would gave been to write: `if (declared && get(desc) && !IsUndefined()`

Reply:
Updated. Thanks for the review!